rfd/0037-desktop-access-protocol.md

```diff
@@ -200,15 +210,37 @@ This message contains a mouse wheel update. Sent from client to server.
 #### 9 - error
 
 ```
-| message type (9) | message_length uint32 | message []byte
+| message type (9) | message_length uint32 | message []byte |
 ```
 
 This message indicates an error has occurred.
 
+#### 28 - notification
+
+```
+| message type (28) | message_length uint32 | message []byte | severity byte |
+```
+
+This message sends a notification message with a severity level. Sent from server to client.
```

Comment:
```suggestion
This message sends a notification message with a severity level. Sent from Teleport to browser.
```

I don't like using server client here, because it's confusing. The "server" in this case is Teleport, but when it comes to RDP Teleport is the RDP client..

Reply:
I don't agree that RDP is relevant here because this is a TDP specification document. It means "sent from TDP server to TDP client".